packages/graph/hash_graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -89,7 +90,7 @@ struct CreateEntityRequest {
     request_body = CreateEntityRequest,
     tag = "Entity",
     responses(
-        (status = 201, content_type = "application/json", description = "The created entity", body = PersistedEntityIdentifier),
+        (status = 201, content_type = "application/json", description = "The created entity", body = PersistedEntityMetadata),
```

Comment:
This description has been incorrect for a long time for example 🙃

Reply:
Good spot, addressed in https://github.com/hashintel/hash/pull/1199/commits/94aead455e8faee24ee583ffe0b006c387147a33